lib/generators/sufia/templates/catalog_controller.rb

```diff
@@ -80,6 +80,8 @@ def self.modified_field
     config.add_index_field solr_name("resource_type", :stored_searchable), label: "Resource Type", link_to_search: solr_name("resource_type", :facetable)
     config.add_index_field solr_name("file_format", :stored_searchable), label: "File Format", link_to_search: solr_name("file_format", :facetable)
     config.add_index_field solr_name("identifier", :stored_searchable), label: "Identifier", helper_method: :index_field_link, field_name: 'identifier'
+    config.add_index_field solr_name("embargo_release_date", Solrizer::Descriptor.new(:date, :stored, :indexed)), label: "Embargo release date", helper_method: :human_readable_date
```

Comment:
The field goes missing from the index

![screen shot 2016-08-10 at 3 32 06 pm](https://cloud.githubusercontent.com/assets/4163828/17568064/161cea90-5f10-11e6-9318-580896157193.png)